solutions/dedicated/variables.tf

```diff
@@ -0,0 +1,67 @@
+variable "ibmcloud_api_key" {
+  description = "The IBM Cloud API key to deploy IAM-enabled resources."
+  type        = string
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "The IBM Cloud region where the instance of IBM Cloudant is provisioned."
+  default     = "us-south"
+}
+
+variable "existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group. If false, specify the name for the new resource group in the `resource_group_name` input."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "Add a new resource group with this name if the `existing_resource_group` input is false. To use an existing resource group, set `existing_resource_group` to true instead."
```

Comment:
This description is inaccurate. This value is required regardless of whether `existing_resource_group` os true or false. Suggest to change to:
"The name of a new or an existing resource group in which to provision the Cloudant instance in."